changes/note_#2_GLT-3911.md

```diff
@@ -0,0 +1 @@
+Links for external projects that use their short names changed from (`http://example.com/project/{shortName}`) to using their code (`http://example.com/project/{code}`)
```

Comment:
This is not explaining what needs to be done like your other note is. Something like "If your `miso.properties` file specifies `miso.project.report.links`, any `{shortName}` placeholders in this must be updated to `{code}`"